### PR TITLE
Lesson3 homework

### PR DESCRIPTION
**Pull request by deb-lab**
Hello.
Seems I'm complete some tasks and ready to code review. 
Thanks.

### CODE REVIEW

File: Hrebenevych/HW3/task1.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# encoding: utf-8
+
+a = 10
+b = 20
```
Comment (deb-lab) [inline Hrebenevych/HW3/task1.py:4]: Little problems with naming variables.

File: Hrebenevych/HW3/task4.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        variables[i] = input(f'Enter the {i} of circle: ').strip()
+
+        try:
+            variables[i] = float(variables[i])
+            successfull = True
+        except:
+            print(f"{i.capitalize()} must have a integer or float value. Delimiter is '.'")
+            continue
+
+pi = 3.14159265359
```
Comment (katemalash) [inline Hrebenevych/HW3/task4.py:23]: Is it possible to use another way?

File: Hrebenevych/HW3/task2.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    if size.isdigit():
+        size = int(size)
+        break
+    else:
+        print()
+        print('Please enter correct numerical value.')
+        print()
+
+
+kilobytes = size / 1000
```
Comment (katemalash) [inline Hrebenevych/HW3/task2.py:15]: Are you sure you use the correct calculations?

**Review by katemalash — CHANGES_REQUESTED** [review]
Please fix the task 2

File: Hrebenevych/HW3/task4.py (hunk 2 above)
Comment (deb-lab) [reply Hrebenevych/HW3/task4.py:23]: We can use `math.pi` from the math module.

File: Hrebenevych/HW3/task5.py [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+import math
+
+
+print('Simple program for calculating area the length of the hypotenuse.')
+
+
+variables = {
+        'first': None,
+        'two': None
```
[the remark this answers is not in the document]
Comment (deb-lab) [reply Hrebenevych/HW3/task5.py:11]: Changed. Thanks.

File: Hrebenevych/HW3/task2.py (hunk 3 above)
Comment (deb-lab) [reply Hrebenevych/HW3/task2.py:15]: Changed to 1024.

**Review by katemalash — APPROVED** [review]
Looks good now. You can merge your pull-request

**Pull request merged**